.travis.yml

```diff
@@ -28,7 +28,7 @@ stages:
 jobs:
   include:
     - stage: basic tests
-      script: pytest icepyx/ --ignore icepyx/tests/behind_NSIDC_API_login.py
+      script: pytest icepyx/ --username=$EARTHDATA_USERNAME --password=$EARTHDATA_PASSWORD --email=$EARTHDATA_EMAIL --ignore icepyx/tests/behind_NSIDC_API_login.py
```

Comment:
This strikes me as another option for providing login info for integrated testing, similar to the one used below in `- stage: behind Earthdata` (where NSIDC_LOGIN is saved within Travis, which currently runs our pytest suite). This first suite of tests explicitly avoids any functions that are behind the Earthdata login so that they're tested less frequently (and thus ping NSIDC less frequently). But I also see the need to test all of our methods of logging in. I'll need to revisit our existing setup to refresh my memory on the details of how it's set up.

Reply:
good point.  Could easily not have these checks to ping NSIDC.  The included login test is pretty slim to really just check credentials.  Happy to help think this out on the next team call.

Reply:
Are you available for the next team call (24 May at 8 UTC)? Can we plan to spend some time on testing per the comments in this PR?

Reply:
I'll be there :+1: 
Sounds good.  Anytime.

Reply:
great - I will be there too!